<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashConverterDialog</class>
 <widget class="QDialog" name="HashConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Title of the dialogbox">Hash / Name converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="hashEdit"/>
   </item>
   <item row="7" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QProgressBar" name="renameProgress">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string comment="File hash rename progress bar tooltip">Progress</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string comment="Button to add files to rename list">Add</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="renameButton">
     <property name="text">
      <string comment="Button to rename files in rename list">Rename files</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="hashToNameButton">
     <property name="text">
      <string comment="Button text">Hash &gt; Name</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="nameToHashButton">
     <property name="text">
      <string comment="Button text">Name &gt; Hash</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hashLabel">
     <property name="text">
      <string comment="Label next to hash editfield">Hash:</string>
     </property>
     <property name="buddy">
      <cstring>hashEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string comment="Button text">Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string comment="Button to remove files to rename list">Remove</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string comment="Label next to name editfield">Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="6" colspan="2">
    <widget class="QTableView" name="filesToRenameTable">
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>16</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="filesToRenameLabel">
     <property name="text">
      <string>Files to rename: (Files will be copied to target name, original files are not removed)</string>
     </property>
     <property name="buddy">
      <cstring>filesToRenameTable</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>nameToHashButton</tabstop>
  <tabstop>hashEdit</tabstop>
  <tabstop>hashToNameButton</tabstop>
  <tabstop>filesToRenameTable</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>renameButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HashConverterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>16</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>nameToHashButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hashEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>hashToNameButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
